cognite_toolkit/cdf_tk/templates.py

```diff
@@ -584,6 +586,8 @@ def _dump_yaml_with_comments(
     for line in dumped.splitlines():
         indent = len(line) - len(line.lstrip())
         if last_indent < indent:
+            if last_variable is None:
+                raise ValueError("Unexpected state of last_variable being None")
```

Comment:
If this actually happens, this is a bad error message.

Reply:
It cannot happen, just for `mypy`